devices/popp.js

```diff
@@ -66,7 +67,9 @@ module.exports = [
                 .withDescription('Mean radiator load for room calculated by gateway for load balancing purposes (-8000=undefined)')
                 .withValueMin(-8000).withValueMax(100),
             exposes.numeric('load_estimate', ea.STATE_GET)
-                .withDescription('Load estimate on this radiator')],
+                .withDescription('Load estimate on this radiator'),
+            exposes.enum('programming_operation_mode', ea.ALL, ['setpoint', 'schedule'])
```

Comment:
Please move this expose to `exposes.js`, could you also improve the description?

Reply:
> Please move this expose to exposes.js

As its own thing or as another `withXyz` on the climate expose?

> could you also improve the description

Sure, but it will be a little wordy.

Reply:
> As its own thing or as another withXyz on the climate expose?

Own thing

> Sure, but it will be a little wordy.

No problem, no need to keep things short there.